src/styles/blocks/catalog.scss

```diff
@@ -1,17 +1,17 @@
 .catalog {
   display: flex;
   flex-wrap: wrap;
-
   max-width: 944px;
   margin: 0 auto;
   padding: 50px 0;
 
+  $min-time: 300ms;
```

Comment:
Move this variable to the separate _variables.scss file and use it in catalog.scss and header.scss